controllers/api.controller.js

```diff
@@ -0,0 +1,19 @@
+const endpointsJson = require("../endpoints.json");
+const {getAllTopics} = require ('../models/api.models')
+
+function getApi(req, res) {
+    console.log(endpointsJson, "<--- endpoints json");
+    res.status(200).send({endpoints: endpointsJson})
+}
+function getApiTopics (req, res){
+    getAllTopics()
+    .then((topics) => {
```

Comment:
The names here could be slightly more semantic, for example, "getTopics" is the controller function and "fetchTopics" or "selectTopics" would be more appropriate for the model function. The function looks good though. Nice work.